tests/previousresults/getrequest_public-:collectionId-:displayId(*)-:version-full_.html

```diff
@@ -320,39 +326,39 @@ <h4 class="panel-title">
            <tbody>
             <tr>
              <td>
-              synbiohub#topLevel
-              <a href="http://wiki.synbiohub.org/wiki/Terms/synbiohub#topLevel" title="Learn more about this annotation type">
+              synbiohub#ownedBy
```

Comment:
Is this a test reordering? We need to be careful checking in such changes (also need to fix them.)